src/OpenConext/ProfileBundle/Service/GlobalViewParameters.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+/**
+ * Copyright 2015 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace OpenConext\ProfileBundle\Service;
+
+use InvalidArgumentException;
+use OpenConext\Profile\Assert;
+use Symfony\Component\Translation\DataCollectorTranslator;
+
+class GlobalViewParameters
+{
+    /**
+     * @var DataCollectorTranslator
+     */
+    private $translator;
+
+    /**
+     * @var array
+     */
+    private $helpUrls;
+
+    /**
+     * @var array
+     */
+    private $platformUrls;
+    /**
+     * @var array
+     */
+    private $termsOfServiceUrls;
+
+    /**
+     * @param DataCollectorTranslator $translator
+     * @param array $helpUrls
+     * @param array $platformUrls
+     * @param array $termsOfServiceUrls
+     */
+    private function __construct(
+        DataCollectorTranslator $translator,
```

Comment:
Why this specific implementation? I guess this may actually fail in production, since it used for the web profiler.